src/ripple/app/ledger/impl/LedgerCleaner.cpp

```diff
@@ -301,7 +311,8 @@ class LedgerCleanerImp
 
         if(! getApp().getLedgerMaster().fixIndex(ledgerIndex, ledgerHash))
         {
-            m_journal.debug << "ledger " << ledgerIndex << " had wrong entry in history";
+            m_journal.debug << "ledger " << ledgerIndex
+                            << " had wrong entry in history";
```

Comment:
I think most ripple code puts the operator `<<` at the end of the previous line and indents the next line 4. Tho I don't think there's an official style guide for this.